charts/hivemq-platform/values.yaml

```diff
@@ -350,10 +350,10 @@ services:
 # Note: When overriding default configurations, make sure to include required options into your
 #       custom configurations. See the documentation for more.
 config:
-  # Create a default HiveMQ Platform configuration (config.xml) as a Kubernetes ConfigMap.
+  # Creates a default HiveMQ Platform configuration (config.xml) as a Kubernetes ConfigMap by default.
   create: true
```

Comment:
Suggestion: place `createAs: (secret|configmap)` , right after the `create` variable. (Instead of `useSecret` at the bottom`.

Reply:
Done, please have another look